README.md

```diff
@@ -1,30 +1,35 @@
-After one hour, the access_token needs to be refreshed by the ‘Refresh tokens’ endpoint. Simply send it and the variables holding the tokens will be overwritten with the new tokens.
+## Collection variables
+Most endpoint payloads contain variables instead of static values. All variables are initiated at the collection level, and contain dummy data that you can easily adjust. The saved examples display the actual values.
 
-## TESTMODE
-Keep in mind to use the collections with an API key for test mode. The collection based on OAuth/access_tokens uses the test mode query-parameter (GET/DELETE) or payload-parameter (PUT/PATCH) by default.
+## Credentials
+The JSON files contain variables instead of credentials. Postman supports authentication inheritance, so the variables are often called on higher-level objects (collection or folder). Best practice is to create a Postman Environment and initiate all credentials there to prevent credentials from appearing in the collection itself. Besides, you can easily create multiple environments and use them to switch between Mollie accounts or App credentials.
 
-## FEEDBACK
-Mollie will keep the collection aligned with the public API documentation. Feedback or additional examples are welcomed and can be submitted as a comment or PR.
+## OAuth
+To retrieve the access tokens of another account using OAuth, use the endpoints in the Connect folder of the collection that utilises OAuth, that is, access tokens:
+
+1. Go to **Developers** > **Your apps** in your Mollie Dashboard and register an app in the “partner” account (the one that takes control).
+2. Place the client ID and secret in the authorization settings ‘OAuth API’ folder or in the environment variables.
+3. Send the ‘Authorise’ request. This constructs the OAuth URL that you can fetch from the Postman console.
+4. Log in to the “submerchant” Mollie account (the account that is controlled) and navigate to the URL in your preferred browser.
+5. Connect the accounts, and retrieve the `code` parameter from the landing page URL. If you were not logged in to Mollie, you can do so here.
+6. Paste the copied code in the `code` parameter in the payload of the ‘Generate access tokens’ request in the same collection folder, and send it. This should take no longer than 30 seconds. If the code expires, go back to step 4.
+7. The Postman tests read Mollie’s response and assign the tokens to global variables for further use. You can now send requests to all other endpoints in the collection and control the “submerchant” account.
+
+After one hour, you need to refresh the `access_token` using the ‘Refresh tokens’ endpoint. Send a request to the endpoint, and the variables holding the tokens will be overwritten with the new values.
+
+## Test mode
+Use the collections with an API key for test mode. **Mollie v2 API - OAuth-access_token authentication** uses the test mode query parameter (`GET`/`DELETE`) or payload-parameter (`PUT`/`PATCH`) by default.
```

Comment:
"Use the collections with an API key for test mode" --> ambiguous, does "for" belong to "Use the collections with an API key" or "an API key"

Reply:
```suggestion
For test mode, use **Mollie v2 API - API Key authentication** with a test API key. **Mollie v2 API - OAuth-access_token authentication** uses the test mode query parameter (`GET`/`DELETE`) or payload-parameter (`PUT`/`PATCH`) by default.
```